packages/firestore/exp/test/shim.ts

```diff
@@ -168,7 +169,7 @@ export class FirebaseFirestore implements legacy.FirebaseFirestore {
 
   loadBundle(
     bundleData: ArrayBuffer | ReadableStream<Uint8Array> | string
-  ): legacy.LoadBundleTask {
+  ): LoadBundleTask {
```

Comment:
This should return the legacy Task (all APIs return the legacy type), even though it is for now the same.

Reply:
Done.